mypy/semanal_shared.py

```diff
@@ -37,9 +37,10 @@ def lookup_qualified(self, name: str, ctx: Context,
                          suppress_errors: bool = False) -> Optional[SymbolTableNode]:
         raise NotImplementedError
 
-    @abstractmethod
     def lookup_fully_qualified(self, name: str) -> SymbolTableNode:
-        raise NotImplementedError
+        ret = self.lookup_fully_qualified_or_none(name)
+        assert ret is not None
+        return ret
```

Comment:
Similar to above, maybe we can keep this abstract if the simple implementation is in `mypy.semanal`.